src/js/pages/jobs/JobsTab.js

```diff
@@ -13,61 +7,118 @@ import FilterHeadline from "../../components/FilterHeadline";
 import JobsBreadcrumbs from "../../components/breadcrumbs/JobsBreadcrumbs";
 import JobsTable from "./JobsTable";
 import JobSearchFilter from "../../components/JobSearchFilter";
-import JobFilterTypes from "../../constants/JobFilterTypes";
 import JobFormModal from "../../components/modals/JobFormModal";
 import JobTree from "../../structs/JobTree";
 import Loader from "../../components/Loader";
 import Page from "../../components/Page";
-import ServiceFilterTypes from "../../../../plugins/services/src/js/constants/ServiceFilterTypes";
 
-const METHODS_TO_BIND = [
-  "getHeadline",
-  "handleFilterChange",
-  "handleCloseJobFormModal",
-  "handleOpenJobFormModal",
-  "resetFilter",
-  "resetFilterQueryParams"
-];
+const METHODS_TO_BIND = ["handleCloseJobFormModal", "handleOpenJobFormModal"];
+
+const ConditionalFilterHeadline = ({
+  item,
+  filteredJobs,
+  hasFilterApplied,
+  resetFilter
+}) => {
+  const jobs = item.getItems();
+
+  if (hasFilterApplied) {
+    return (
+      <FilterHeadline
+        onReset={resetFilter}
+        name="Jobs"
+        currentLength={filteredJobs.length}
+        totalLength={jobs.length}
+      />
+    );
+  }
+
+  return null;
+};
 
-var DEFAULT_FILTER_OPTIONS = {
-  searchString: ""
+const JobsTabLoading = ({ root }) => {
+  return (
+    <Page>
+      <Page.Header breadcrumbs={<JobsBreadcrumbs tree={root} />} />
+      <Loader />
+    </Page>
+  );
 };
 
-class JobsTab extends mixin(StoreMixin) {
+const JobsTabList = ({
+  item,
+  root,
+  modal,
+  filteredJobs,
+  searchString,
+  handleFilterChange,
+  handleOpenJobFormModal,
+  resetFilter,
+  hasFilterApplied
+}) => {
+  return (
+    <Page>
+      <Page.Header
+        addButton={{
+          label: "Create a job",
+          onItemSelect: handleOpenJobFormModal
+        }}
+        breadcrumbs={<JobsBreadcrumbs tree={root} item={item} />}
+      />
+      <div className="flex-grow">
+        <ConditionalFilterHeadline
```

Comment:
We could simply have lift up the `hasFilterApplied` logic to the `JobsTabList` and keep `FilterHeadline` plain no logic dumb